src/styles/header.scss

```
@@ -0,0 +1,66 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+
+  &__image {
+    height: 100%;
+    width: 100%;
+  }
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
```

just FYI, you can also do it in this way:

```suggestion
.nav__item:not(:first-child) {
```